auto_round/quantizer.py

```diff
@@ -231,18 +233,39 @@ def __init__(self, orig_layer, enable_minmax_tuning=True):
         else:
             self.min_scale = torch.tensor(1.0, device=self.orig_layer.weight.device, dtype=weight_dtype)
             self.max_scale = torch.tensor(1.0, device=self.orig_layer.weight.device, dtype=weight_dtype)
+        
+        self.enable_teq = enable_teq
+        if self.enable_teq:
+            from auto_round import scale
+            self.weight_scale_calculator = scale.ScaleCalculator(self.orig_layer.weight.data, self.orig_layer.weight.device)
```

Comment:
The current implementation confuses me ,  I don't konw which op goes first, teq -> rounding or rounding ->teq